internal/pkg/aws/secretsmanager/secretsmanager.go

```diff
@@ -91,3 +111,13 @@ type ErrSecretAlreadyExists struct {
 func (err *ErrSecretAlreadyExists) Error() string {
 	return fmt.Sprintf("secret %s already exists", err.secretName)
 }
+
+// ErrSecretNotFound occurs if a secret with the given name does not exist.
+type ErrSecretNotFound struct {
+	secretName string
+	parentErr  error
```

Comment:
nit: looks like we don't use this field, should we remove it?

Reply:
We use it in `pipeline_delete.go`.... Will look into streamlining this after this push! 😸